test_pytest_timeout.py

```diff
@@ -603,3 +603,22 @@ def test_foo():
             "pytest_timeout_cancel_timer",
         ]
     )
+
+def test_suite_timeout(pytester):
+    pytester.makepyfile(
+        """
+     import time, pytest
+
+     @pytest.mark.parametrize('i', range(10))
+     def test_foo(i):
+         time.sleep(0.1)
+    """
+    )
+    # each parametrization runs for 0.1 sec
+    # or about 0.00166 seconds each
```

Comment:
minutes?  but why make the maths complicated?  why not sleep for 1 second?

Trying to make this test suite fast just makes it flaky, make the timeouts used comfortably large, never smaller than 1s.  So I'd aim for a "suite timeout" of 1 second here.

Reply:
Yea. Seconds is more consistent with the rest of the timeouts.

Reply:
And letting the test suite run for a second seems ok.